src/js/utils/permissionUtils.js

```diff
@@ -0,0 +1,45 @@
+import ActivityCode from 'consts/activityCode';
+import RequisitionStatus from 'consts/requisitionStatus';
+import { supports } from 'utils/supportedActivitiesUtils';
+
+const canEditRequestWithoutRequiredApproval = (location, origin, destination, isUserRequestor) => {
```

Comment:
What is the advantage of putting this in a separate function? I feel like this one could be directly in the `canEditRequest`. Plus looks like you are also using `isLocationOrigin` and `isLocationDestination` in both functions, so these consts could be moved to the top of the `canEditRequest` util and used when needed.